lib/db.ts

```diff
@@ -176,3 +178,13 @@ export async function updateCheckin(
   const checkin = await getCheckin(id)
   return checkin
 }
+
+export async function setAcceptedPrivacy(companyId: string): Promise<Guest> {
+  let guest = await db.guests.toCollection().first()
```

Comment:
You can use `getGuest()` gere

```suggestion
  let guest = await getGuest()
```